sufia-models/app/models/sufia/works/curation_concern/with_generic_files.rb

```diff
@@ -0,0 +1,17 @@
+module Sufia::Works
+  module CurationConcern
+    module WithGenericFiles
+      extend ActiveSupport::Concern
+
+      included do
+        # This used to have a hasFile relation when in hydra-works.  That does not seem to exist so I am using hasPart instead
```

Comment:
There are already tickets for changing the object to PCDM (#1007).  I figured that was enough...